ec2/spark_ec2.py

```diff
@@ -178,6 +178,9 @@ def parse_args():
         "-i", "--identity-file",
         help="SSH private key file to use for logging into instances")
     parser.add_option(
+        "-p", "--profile", default="default",
+        help=" If you have multiple profiles, you can configure additional, named profiles by using this option (default: %default)")
```

Comment:
Actually speaking, we can use any types of profile(AWS/boto). The remaining problem is just the loading order of these "profiles" as [Jan Vlcinsky explained](https://stackoverflow.com/questions/11286479/recommended-way-to-manage-credentials-with-multiple-aws-accounts). I think that it is little bit long to explain that in commandline help.